components/datetime/src/provider/calendar/mod.rs

```diff
@@ -30,7 +32,9 @@ pub struct DatePatternsV1<'data> {
     pub length_combinations: patterns::GenericLengthPatternsV1<'data>,
 }
 
-#[icu_provider::data_struct(TimePatternsV1Marker = "datetime/timelengths@1")]
+#[icu_provider::data_struct(
+    marker(TimePatternsV1Marker, "datetime/timelengths@1", extension_key = "ca")
```

Comment:
question: should this go away after https://github.com/unicode-org/icu4x/issues/2153?

Reply:
yes.